Tests/FuzzilliTests/InliningTest.swift

```diff
@@ -63,6 +63,9 @@ class InliningTests: XCTestCase {
 
         let referenceProgram = b.finalize()
 
+        // Delete type information to compare only code
+        referenceProgram.types = ProgramTypes()
```

Comment:
How about doing `XCTAssertEqual(inlinedProgram.code, referenceProgram.code)` and defining the comparison for `Code` in the same way as for the program (i.e. via the protobuf?). Would that work? That seems like what this test really wants to accomplish

Reply:
yeah well we have to define how to convert `Code` to protobuf and define equality opertor